public/app/plugins/datasource/cloudwatch/migrations/variableQueryMigrations.test.ts

```diff
@@ -35,6 +35,19 @@ describe('variableQueryMigrations', () => {
           expect(query.dimensionFilters).toStrictEqual({});
         });
       });
+      describe('and filter param is empty array', () => {
```

Comment:
nitpicky nonblocking suggestion: is there a way to describe `filter param is empty` in a way that is more user-focused so it's a little more clear what this is describing?

Reply:
changed it a little, but short of putting `[]` in the test name, i wasn't really sure how to describe it